### ISSUE TITLE
Picture 3 and 4

### ISSUE THREAD

**Issue by damse0045** [damse0045/Universal-bitcoin-value-color-scale.#2]
![capture3](https://user-images.githubusercontent.com/16904388/44309640-183a5f00-a3ca-11e8-91b4-a22ea316d506.JPG)
![capture4](https://user-images.githubusercontent.com/16904388/44309642-1cff1300-a3ca-11e8-930a-a33a1136eeba.JPG)

**Comment by odarboe** [comment]
@damse0045 I think you should avoid using the yellow color due to visibility issues when looking at the price for Picture 3 and 4.

**Comment by odarboe** [comment]
@damse0045 How about using a single color palette such as 
![bild](https://user-images.githubusercontent.com/1871493/44342970-daead580-a48c-11e8-8340-c922461e7017.png)

**or** 

![bild](https://user-images.githubusercontent.com/1871493/44343006-f8b83a80-a48c-11e8-8479-f63b09910b03.png)

Source: https://uxplanet.org/designing-systematic-colors-b5d2605b15c

**Comment by damse0045** [comment]
@odarboe yes, you are right. Yellow is bad :) And thank you for the link. I have not read the whole paper just yet, however your illustration and line of thinking is much in line with a number of other comments I have received. The beautiful thing is, that I think it is possible to combine the different ideas into a better proposal. Here is a suggestion from a Wouter Schut, which fits with your color theme:
![image](https://user-images.githubusercontent.com/16904388/44346095-68cabe80-a495-11e8-8299-569f92391a67.png)


It is good to feel the common Interest for making a better bitcoin, and a common understanding for the issue. Again thank :)

**Comment by StopAndDecrypt** [comment]
I've seen a lot of varying opinions on this subject, and I'd like to highlight some common talking points.

1. This would be an optional standard, as such, services _opt_ to use it, and they create a means for people to _opt_ out of using it. Those who are colorblind already don't have an option, and this extra option takes nothing away from them. With that being said, it would be a well designed feature if they are taken into consideration.

2. Putting colorblind people to the side just to make this point: Rainbows are terrible for distinguishing place value, and gradients of a single color are worse.

3. The color visibility is dependent on the contrast it creates with the background it's layered over. We're not trying to standardized a white or black background, so we shouldn't be picking a color that works with both.

4. There is no need for each place value to have it's own color. At most they should be divided and colored by sections, the way many are already used to interpreting large numbers, like Wouter's example above, or this graphic I made below (that I'm not too fond of myself):

![untitled](https://user-images.githubusercontent.com/37635933/44492227-b714c480-a631-11e8-98cf-35b921665b48.png)

5. We're all familiar with bits (uBTC), as well as millibitcoin (mBTC). Regardless of your opinions on the naming conventions, the place values are widely used, and any color(s) that are chosen should respect the u/mBTC denominations. I've seen some people not respect this, where they try to divide them differently, which to me just seems really odd: 21 , 000 , 000 . 00 , 000 , 000

Pulling all of this together, these are my thoughts:

A single color should be used, and that color should be blue. It is the most accessible to those who are colorblind.

By dividing the right of the decimal into satoshis/not-satoshis, you're left with the two aforementioned groups of units (m/uBTC) separated by a single comma. Visual assistance beyond that comes with too many trade-offs.

I like Wouter's concept and the standard can allow services to create a colorblind theme that respects the blue while also creating their own theme that respects the rest of the format, but gives them the freedom to choose their own color scheme for their wallet. They can do a red-on-black theme, purple-on-gold, etc., and it won't matter. They could even go the extra mile and create a secondary colorblind theme for those with Tritanopia (difficulty distinguishing between blue & yellow, but red stands out).

Maybe even the "0."'s can be bold:

![bold](https://user-images.githubusercontent.com/37635933/44493109-cf3a1300-a634-11e8-935e-e94d7d6291b2.png)

**Comment by damse0045** [comment]
First thanks for the good and well thought comments, @StopAndDecrypt. Secondly, let’s move away from my rainbow-suggestion. I accept the different arguments, and acknowledge in hindsight that the suggestion was not thought thoroughly enough through. So rainbows are out of the question.

I also like Wouter’s concept and if I read your comment correctly, that is also the essence of your what you write. So in that sense everything is just great and kumbhaka. 

But, it still irritates me that I cannot come up with a better solution, in the sense that I personally have a hard time believing that I in the further will be able to decipher a sales sign:
![price](https://user-images.githubusercontent.com/16904388/44516321-1374f400-a6c5-11e8-9210-b383e37ae9f3.JPG)



in the same effortless way I read such signs today. 
So therefore I know I personally will have to play around with this some more and maybe I will end up with another hopeless and weird version 0.2, that then have overlooked some next basis element. I hope you also at that time will help me understand what I have overlooked :).
But right now Wouter’s concept is the best one. Again thank for the comments
